frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -302,9 +302,14 @@ describe("collection permissions", () => {
                 });
 
                 it("should be able to archive the question (metabase#11719-3)", () => {
```

Comment:
Feel free to add the reference to #16512 as well.

Reply:
Does `(metabase#11719-3, metabase#16512)` look good?